### ISSUE TITLE
Expected input dimension for outcome nuisance model in DML

### ISSUE THREAD

**Issue by divyat09** [microsoft/EconML#564]
I want to use the fitted outcome model (E[Y | X, W]) from the DML estimator and validate it on a new dataset. I can access the outcome model from the DML object (estimator) using estimator.models_y. Specifically, estimator.models_y is a LinearRegression model from sklearn.  

But I get an error when I try to obtain predictions using it on a new dataset, it does not expect the input dimensions to be dim(X)+dim(W). I was under the impression that the input is the concatenation of X and W, so the input dimensions should be the sum of the two.

**Comment by kbattocchi** [comment]
Sorry for the late reply, this was also asked in #532.  The dimensions of the outcome model depend on how `linear_first_stages` is set: if it is False, then as you expect the input is just [X;W].  However, as explained in the linked issue if `linear_first_stages` is True then we use [X,W] cross [1;phi(X)], where phi is the featurizer used (by default the identity).

**Comment by divyat09** [comment]
Thanks, this worked. Sorry I had forgotten to close the issue earlier.
